src/cli/cli.cpp

```diff
@@ -7573,6 +7589,23 @@ template <> otError Interpreter::Process<Cmd("timeinqueue")>(Arg aArgs[])
 
 template <> otError Interpreter::Process<Cmd("dataset")>(Arg aArgs[]) { return mDataset.Process(aArgs); }
 
+/**
+ * @cli txpower(get,set)
+ * @code
```

Comment:
Can you swap the examples so the set is first (so it looks like you're setting it to -10, then getting the -10 back)